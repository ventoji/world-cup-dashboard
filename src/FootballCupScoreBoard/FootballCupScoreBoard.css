.main-content {
  display: flex;
  flex-wrap: wrap;
  flex-direction: column;
  margin: 30px;
}

.show-matches {
  display: flex;
}
.scoreboard {
  border: 1px solid black;
  padding: 20px;
  margin: 2px;
  flex-grow: 2;
}

.summary-result {
  border: 1px solid black;
  padding: 20px;
  margin: 2px;
  flex-grow: 1;
}

.add-players {
  border: 1px solid black;
  padding: 20px;
  margin: 2px;
}
