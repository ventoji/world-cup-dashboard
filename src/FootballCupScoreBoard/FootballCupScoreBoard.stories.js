
// import React from 'react';
// import FootballCupScoreBoard from './FootballCupScoreBoard';

// export default {
//   title: 'WC App/FootballCupScoreBoard',
//   component: FootballCupScoreBoard
// };


// const Template = (args) => <FootballCupScoreBoard {...args} />;

// export const Default = Template.bind({});

