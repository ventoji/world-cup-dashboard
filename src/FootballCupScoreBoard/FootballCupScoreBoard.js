import React from 'react';
import './FootballCupScoreBoard.css';
import StartGameWCConnected from '../StartGameWC/StartGameWCConnected';
import ScoreBoardConnected from '../ScoreBoard/ScoreBoardConnected';
import SummaryResultWCConnected from '../SummaryResult/SummaryResultsConnected';

const FootballCupScoreBoard = () => {
  return (
    <>
      <main className="main-content">
        <div className="show-matches">
          <div className="scoreboard">
            <h4> list of active games </h4>
            <ScoreBoardConnected />
          </div>
          <div className="summary-result">
            <h4> Summary of the ended games </h4>
            <SummaryResultWCConnected />
          </div>
        </div>

        <div className="add-players">
          <h4> Start a new game </h4>
          <StartGameWCConnected />
        </div>
      </main>
    </>
  );
};

export default FootballCupScoreBoard;
