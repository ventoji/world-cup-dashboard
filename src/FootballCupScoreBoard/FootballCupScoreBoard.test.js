import React from 'react';
import { shallow } from 'enzyme';
import toJson from 'enzyme-to-json';
import FootballCupScoreBoard from './FootballCupScoreBoard';


describe('FootballCupScoreBoard  Component', () => {

    let footballComponent;
   
    
    beforeEach(() => {
        footballComponent = shallow(
        <FootballCupScoreBoard />,
      );
    });

    it('Should be defined', () => {
        expect(FootballCupScoreBoard).toBeDefined();
      });

    
  it('Should render properly', () => {
  

    expect(toJson( footballComponent)).toMatchSnapshot();
 
  });


})