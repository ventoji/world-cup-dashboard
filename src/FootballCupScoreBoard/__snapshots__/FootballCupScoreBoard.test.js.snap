// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`FootballCupScoreBoard  Component Should render properly 1`] = `
<Fragment>
  <main
    className="main-content"
  >
    <div
      className="show-matches"
    >
      <div
        className="scoreboard"
      >
        <h4>
           list of active games 
        </h4>
        <Connect(ScoreBoardConnected) />
      </div>
      <div
        className="summary-result"
      >
        <h4>
           Summary of the ended games 
        </h4>
        <Connect(SummaryResultWCConnected) />
      </div>
    </div>
    <div
      className="add-players"
    >
      <h4>
         Start a new game 
      </h4>
      <Connect(StartGameWCConnected) />
    </div>
  </main>
</Fragment>
`;
